source/dotnet/Library/AdaptiveCards/AdaptiveCardConverter.cs

```diff
@@ -62,16 +62,21 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
 
             if (reader.Depth == 0)
             {
-                // Needed for ID collision detection after fallback was introduced
-                ParseContext.Initialize();
-
                 ValidateJsonVersion(ref jObject);
 
                 if (new AdaptiveSchemaVersion(jObject.Value<string>("version")) > AdaptiveCard.KnownSchemaVersion)
                 {
                     return MakeFallbackTextCard(jObject);
                 }
             }
+
+            /// this is needed when client calls Deserailizer method, we need this contract resolver,
```

Comment:
>Deserailizer [](start = 49, length = 12)

nit: typo :)

Reply:
thanks!

---
In reply to: [340735553](https://github.com/microsoft/AdaptiveCards/pull/3548#discussion_r340735553) [](ancestors = 340735553)